Sources/LithiumExample/ViewController.swift

```diff
@@ -25,8 +25,15 @@ class ViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
     @IBAction func onStartCommunity(_ sender: UIButton) {
+        let deviceToken = UserDefaults.standard.string(forKey: "deviceToken")
```

Comment:
you'd probably want to make this a constant.

Reply:
This is just for the demo. Won't be using it anywhere else.